plugins/module_utils/turbo/server.py

```diff
@@ -220,13 +220,62 @@ def run_lookup_plugin(data):
             name=lookup_name, loader=templar._loader, templar=templar
         )
 
-        # run plugin
-        result = instance._run(terms, variables=variables, **kwargs)
+        if not hasattr(instance, "_run"):
+            return [None, "No _run() found"]
+        result = [inspect.iscoroutinefunction(instance._run)]
```

Comment:
I'm not sure I understand why this line is here. This seems like a weird default to set the result to.

Reply:
leftover from debugging. will be removed